.github/workflows/update-release-links.yml

```
@@ -1,8 +1,9 @@
 name: Update release links
 
 on:
+  workflow_dispatch:
   release:
-    types: [published]
+    types: [published, released]
```

```suggestion
    types: [released]
```

If `released` works, let's use it, but we shouldn't listen to two events that might happen from the same action (it would execute the workflow twice).